docs/release-notes/cloud/cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md

```diff
@@ -0,0 +1,24 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Removal of Jira, Slack, and Webhooks repository integrations November 13, 2023
+
+On the week of November 13th 2023 we removed the option to create, configure, and delete Jira, Slack, and Webhooks integrations for your repositories. These integrations were available on the repository **Settings**, tab **Integration**.
+
+<!--TODO CY-7215 Add rationale for this removal-->
```

Comment:
@RaquelHipolito @mrfyda I need your help here... How can we explain to our customers the rationale for removing these integrations?

Reply:
be honest 😁 

these integrations had low adoption, as a consequence their maintenance had been neglected for years, and ultimately they we making shipping new features slower.
that being said, it's not that we don't believe in the problems they were solving, we just weren't happy with the current implementations. while there is no timeline in place to make them available again, we're open to looking into them again